ray-operator/controllers/ray/common/ingress.go

```diff
@@ -87,7 +86,7 @@ func BuildIngressForHeadService(cluster rayv1.RayCluster) (*networkingv1.Ingress
 	// Get ingress class name from rayCluster annotations. this is a required field to use ingress.
 	ingressClassName, ok := cluster.Annotations[IngressClassAnnotationKey]
 	if !ok {
-		logrus.Warn(fmt.Sprintf("ingress class annotation is not set for cluster %s/%s", cluster.Namespace, cluster.Name))
+		log.Info(fmt.Sprintf("ingress class annotation is not set for cluster %s/%s", cluster.Namespace, cluster.Name))
```

Comment:
This `log` variable is initialized at 
https://github.com/ray-project/kuberay/blob/af5d6393cf6cfd7b39bd82e82a65b6aa52fe07fe/ray-operator/controllers/ray/common/pod.go#L37